### DISCUSSION TITLE
Please help me understand how to configure this add-on

### DISCUSSION THREAD

**Discussion by ovizii** [hassio-addons/app-tailscale#345] [Q&A] [answered]
I previously used a different one where UI could configure this TS client machine via the HA add-on configuration page. 

This add-on however says: 

> Configuration
> This add-on has almost no additional configuration options for the add-on itself.
> 
> However, when logging in to Tailscale, you can configure your Tailscale network right from their interface.
> 
> https://login.tailscale.com/

There are very few options I can configure from there, so I was wondering if this interface is completely unusable? It seems to save some settings I change here but not all !? Where would I configure things like "accept routes",  "log level", "source NAT subnet routes" and the likes?

![image](https://github.com/hassio-addons/addon-tailscale/assets/837902/b923bd06-2223-413e-99e5-185f394689a5)

**Comment by lmagyar** [comment, marked as answer]
There are a dozen options, please read the documentation further.

Click on "Show unused optional configuration options" on the Configuration tab.

The add-on's default configuration works "out of the box" to access HA from a tailnet, but you can fine tune anything for a more complex network config.

Note: Be aware that the HA UI behaves a bit strange with optional config options. An unused option won't be grayed out, but will show a usually disabled state. So I suggest to check the settings with the "3 dot menu" -> "Edit in YAML" before saving and restarting just to be sure what is set and what is not.

**Comment by ovizii** [comment]
> Click on "Show unused optional configuration options" on the Configuration tab.

I saw that option and know what options hide behind it. English is not my first language so the documentation I quoted above sounded like one should not use these options but rather configure the tailnet from their interface.

So this means that the screenshot I posted above CAN and should be used to configure this tailscale client and I only misunderstood the documentation?

**Comment by lmagyar** [comment]
Ah, OK, yeah, this should be updated, I think... :) Years ago there were only a few options, but users requred more and more options, and we ended up with a lot of optional settings. The add-on works out-of-the-box as previously, but fine tuning is possible.
